### PR TITLE
#210 - Adding ux-reveal component

### PR DESCRIPTION
**Pull request by fakhrizaki**
Changes to the manifest.json files were ignored in the watch task. That has been fixed!

### CODE REVIEW

**Comment by ivanwills** [comment]
@fakhrizaki, you look like you have added ux-reveal along with the JSON fix (I already have a PR for the JSON fix). Can I suggest you rename this to ux-reveal as that is mostly what it is doing?

File: src/components/ux-reveal/ux-reveal.feature [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Feature: ux-reveal test suite
+
+  Scenario: Loading ux-reveal
+    Given I have loaded component "ux-reveal" with use case "UseCase"
```
Comment (pvshum) [inline src/components/ux-reveal/ux-reveal.feature]: @fakhrizaki Please provide some tests.

File: src/components/ux-reveal/ux-reveal.hbs [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<div id="{{id}}"
+	class="ux-reveal reveal-modal {{#show}}open{{/show}}"
+	style="visibility: {{visibility}}; top: {{top}}px">
+	{{#if isDataModel}}
+		{{{content}}}
+	{{else}}
+    	{{ yield }}
```
Comment (scrypto) [inline src/components/ux-reveal/ux-reveal.hbs:7]: indentation :)

File: src/components/ux-reveal/ux-reveal.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+	},
+	computed: {
+		visibility: function() {
+			return this.get('show') ? 'visible' : 'hidden';
+		},
+		top: function() {
+			if (this.get('show')) {
+				if (this.get('verticalAlignment') === 'center') {
+					// calculate the height of the modal and screen to position it in the center
+					console.info('calc top');
```
Comment (scrypto) [inline src/components/ux-reveal/ux-reveal.js:15]: please remove console.info

File: src/components/ux-reveal/README.md [hunk 4]
```diff
@@ -0,0 +1 @@
+> ux-reveal Documentation
```
Comment (ivanwills) [inline src/components/ux-reveal/README.md:1]: Can you please add link to foundation's reveal

File: src/components/ux-reveal/ux-reveal.js [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+		}
+	},
+	onrender: function() {
+		// register the triggers
+		var triggers = this.get('triggers');
+		if (triggers) {
+			// $(triggers.join(',')).on('click', function() {
+			// 	this.set('show', true);
+			// }.bind(this));
+			console.info('');
```
Comment (scrypto) [inline src/components/ux-reveal/ux-reveal.js:30]: please remove console.info & commented lines above

File: src/components/ux-reveal/ux-reveal.feature [hunk 6]
```diff
@@ -0,0 +1,4 @@
+Feature: ux-reveal test suite
+
+  Scenario: Loading ux-reveal
+    Given I have loaded component "ux-reveal" with use case "dataDriven"
```
Comment (ivanwills) [inline src/components/ux-reveal/ux-reveal.feature:4]: please add a test that at least checks if you have loaded you template

File: src/components/ux-reveal/static/content.html [hunk 7]
```diff
@@ -0,0 +1,4 @@
+<h1>Hi, I am reveal content</h1>
```
Comment (eneo5541) [inline src/components/ux-reveal/static/content.html:1]: This html is unused, isn't it? It should be removed if it is.

**Comment by nathf** [comment]
This PR is quite stale, feel free to re-open when it's more up to date.

**Pull request closed without merging**